languages/csharp/exercises/concept/enums/.docs/instructions.md

```diff
@@ -40,10 +45,10 @@ The log level of a log line is quite verbose. To reduce the disk space needed to
 
 The encoded log level is simple mapping of a log level to a number:
 
-- `Unknown` -> `0`
 - `Info` -> `1`
 - `Warning` -> `2`
-- `Error` -> `4`
+- `MinorError` -> `4`
+- `CriticalError` -> `4`
```

Comment:
Shouldn't these two types of errors have different levels?

Reply:
Yes, you're right. Thanks for pointing it out :)